src/content/docs/en/reference/modules/astro-content.mdx

```diff
@@ -87,7 +88,7 @@ Either an object or a function that allows you to load data from any source, loc
 
 An optional Zod object or function that returns a Zod object to configure the type and shape of document frontmatter for a collection. Each value must use [a Zod validator](https://github.com/colinhacks/zod). (For live collections, see the [live `schema`](#schema-1) property.)
 
-[See the `Content Collection` guide](/en/guides/content-collections/#defining-the-collection-schema) for example usage.
+[See the “Content Collections” guide](/en/guides/content-collections/#defining-the-collection-schema) for example usage.
```

Comment:
Thank you! The backticks in text were bothering me deeply while reviewing this file in the other PR